### PR TITLE
Added expiring feature in user guide

### LINKED ISSUE #70: Update User Guide

### PR DESCRIPTION
**Pull request by kwokyto**

### CODE REVIEW

**Review by SimJJ96 — APPROVED** [review]
LGTM

**Review by Leeyp — DISMISSED** [review]
Would be good if you could add some sample input/output for this section

File: docs/UserGuide.md [hunk 1]
```diff
@@ -11,27 +11,19 @@
...
-
-`todo n/Refactor the User Guide to remove passive voice d/13/04/2020`
+Format: `expiring`
 
 ## FAQ
 
-**Q**: How do I transfer my data to another computer? 
+**Q**: How do I transfer my data to another computer?
 
 **A**: {your answer here}
```
Comment (joohwan58) [inline docs/UserGuide.md]: pull request #67 has the instructions to save

**Review by joohwan58 — DISMISSED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (Vinci-Hu) [reply docs/UserGuide.md]: I have wrote it in my pr for this.

**Review by Vinci-Hu — APPROVED** [review]
LGTM

**Pull request merged**